UnityProject/Assets/Scripts/Player/TeleportUtils.cs

```diff
@@ -48,17 +56,27 @@ orderby name.name
 					nameOfObject = "Spectator";
 				}
 
-				var status = player.PlayerType switch
+				string status = "";
+
+				if (player.IsGhosting)
+				{
+					status = "(Ghost)";
```

Comment:
we don't need to see ghosts in the list. They were omitted from that list in this PR https://github.com/unitystation/unitystation/pull/9969

Reply:
well this now tracks by a mind so this tell someone if someone has ghosted or not

Reply:
so you won't get duplicate entries from the player and the ghost

Reply:
what use does it have to spectate a ghosted player?

Reply:
well, you can look at the list and tell if your favourite antagonist has died

Reply:
that doesn't sound like a good thing to have

Reply:
it's not that bad having that little bit of extra information